SNR/hdf5_rtplot_utils.py

```diff
@@ -190,44 +191,73 @@ def plot_antennas_range_time(antennas_iq_file, antenna_nums=None,
     end_sample
         The last sample in the sequence to plot. Default 70 so ranges 0-69
         will plot.
-    """ 
-
-    reader = BorealisRead(antennas_iq_file, 'antennas_iq', 
-                          borealis_file_structure='array')
-    arrays = reader.arrays
+    plot_directory
+        The directory that generated plots will be saved in. Default '', which
+        will save plots in the same location as the input file.
+    figsize
+        The size of the figure to create, in inches across by inches tall.
+    """
+    basename = os.path.basename(antennas_iq_file)
 
-    (num_records, num_antennas, max_num_sequences, num_samps) = \
-        arrays['data'].shape
+    if plot_directory == '':
+        directory_name = os.path.dirname(antennas_iq_file)
+    elif not os.path.exists(plot_directory):
+        directory_name = os.path.dirname(antennas_iq_file)
+        print(f"Plot directory {plot_directory} does not exist. Using directory {directory_name} instead.")
+    else:
+        directory_name = plot_directory
 
-    basename = os.path.basename(antennas_iq_file)
-    directory_name = os.path.dirname(antennas_iq_file)
     time_of_plot = '.'.join(basename.split('.')[0:6])
 
-    # typically antenna names and antenna indices are the same except 
-    # where certain antennas were skipped in data writing for any reason.
-    if antenna_nums is None:
-        antenna_indices = list(range(0, num_antennas))
-        antenna_names =  list(arrays['antenna_arrays_order'])
+    # Try to guess the correct file structure
+    if "site" in antennas_iq_file:
```

Comment:
I suggest checking against basename instead, what if the path contains 'site' but it's actually an array file?